docs/installation.rst

```diff
@@ -188,3 +188,10 @@ The installation instructions are the same as the :ref:`local-install`
 installation. You will need to use an AWS account with appropriate permissions.
 The first time you run ``buildstock_aws`` it may take several minutes,
 especially over a slower internet connection as it is downloading and building a docker image.
+
+
+Google Cloud Platform
+~~~~~~~~~~~~~~~~~~~~~
+
+TODO: Add any GCP-specific installation instructions
```

Comment:
FYI, on the client side (workstation where you run the script), there's no GCP stuff to "install." You do need to setup the GOOGLE_APPLICATION_CREDENTIALS key. There is setup to do on the GCP side (like creating a repository).

Reply:
Yeah, I think the basic "you need a GCP project and the appropriate credentials" stuff can go here, similar to the AWS section above. (This reminds me that we'll also need to make sure to update setup.py to include any python packages we're using.)

I'm hoping all the GCP-side setup can be done automatically, via Terraform.

Reply:
> This reminds me that we'll also need to make sure to update setup.py to include any python packages we're using.

Yea, earlier revisions of my work added 'google-api-python-client' and 'google-cloud-storage' to 'install_requires', but I ended up not needing those since interaction with GCP's Artifact Registry went through the Docker Client (and use of Docker standard APIs with AR).